### PR TITLE
Updated OneShard documentation.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: 3.6/architecture-deployment-modes-cluster-architecture.md [hunk 1]
```diff
@@ -202,21 +202,17 @@ processing.
...
 ### How to use the OneShard feature?
 
-The OneShard feature is enabled by default. If you use ArangoDB Enterprise
-Edition and if the collections used in the query are eligible, then the
-optimizer rule `cluster-one-shard` is applied automatically. To be eligible,
-all collections in question have to have a single shard only and the
-`distributeShardsLike` property needs to be set to a common collection name
-(except the prototype collection) to ensure that their single shards are placed
-on the same DB-Server. There are multiple ways to achieve this:
+The OneShard feature is enabled by default if you use ArangoDB Enterprise
```
Comment (jsteemann) [inline 3.6/architecture-deployment-modes-cluster-architecture.md:212]:
```suggestion
The OneShard feature is enabled by default if you use the ArangoDB Enterprise
```

File: 3.7/architecture-deployment-modes-cluster-architecture.md [hunk 2]
```diff
@@ -202,21 +202,17 @@ processing.
...
 ### How to use the OneShard feature?
 
-The OneShard feature is enabled by default. If you use ArangoDB Enterprise
-Edition and if the collections used in the query are eligible, then the
-optimizer rule `cluster-one-shard` is applied automatically. To be eligible,
-all collections in question have to have a single shard only and the
-`distributeShardsLike` property needs to be set to a common collection name
-(except the prototype collection) to ensure that their single shards are placed
-on the same DB-Server. There are multiple ways to achieve this:
+The OneShard feature is enabled by default if you use ArangoDB Enterprise
```
Comment (jsteemann) [inline 3.7/architecture-deployment-modes-cluster-architecture.md:212]:
```suggestion
The OneShard feature is enabled by default if you use the ArangoDB Enterprise
```

**Comment by joerg84** [comment]
@jsteemann PTAL

File: 3.8/architecture-deployment-modes-cluster-architecture.md [hunk 3]
```diff
@@ -226,20 +222,12 @@ on the same DB-Server. There are multiple ways to achieve this:
   The `_graphs` system collection will be used for `distributeShardsLike`.
 
 - For individual OneShard databases, set the `sharding` database property to
-  `"single"` to enforce the OneShard conditions for all collections that will be
-  created in it. The `_graphs` system collection will be used for
+  `"single"` to enforce the OneShard condition. The `_graphs` system collection will be used for
```
Comment (Simran-B) [inline 3.8/architecture-deployment-modes-cluster-architecture.md:225]:
```suggestion
  to enforce the OneShard condition. The `_graphs` system collection will be used for
```

**Pull request merged**